scriptworker/constants.py

```diff
@@ -112,6 +112,8 @@
 
     "base_gpg_home_dir": "...",
     "gpg_lockfile": os.path.join(os.getcwd(), "gpg_homedir.lock"),
+    # The path to the git executable.
+    "git_path": None,
```

Comment:
Let's just set this to `"git"`, and then we don't need `guess_git_path`.

Reply:
Do you mean I should use `context['git_path']` instead of `guess_git_path(context)`. It's kind of makes sense (less tests!), just double checking.

Reply:
Yes :) `context.config['git_path']`